pnpm-workspace.yaml

```diff
@@ -6,7 +6,7 @@ catalog:
   typescript: 5.5.3
   "@typescript-eslint/eslint-plugin": 7.16.1
   "@typescript-eslint/parser": 7.16.1
-  "@types/node": 20.14.10
+  "@types/node": 22.10.10
```

Comment:
I don’t think we can use the types for node 22. We need to find the new release of the 20

Reply:
hmm yeah
there is a v20 inside it that is updated, wonder how we acces that https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/v20/dgram.d.ts